articles/aks/node-autoprovision.md

```diff
@@ -64,11 +64,27 @@ NAP is based on the Open Source [Karpenter](https://karpenter.sh) project, and t
     ```
 
 ## Limitations
-
-- Windows and Azure Linux node pools aren't supported yet
-- Kubelet configuration through Node pool configuration is not supported
-- NAP can only be enabled on new clusters currently
-
+### Unsupported features:
+- Windows and Azure Linux node pools
```

Comment:
Should we remove the AzureLinux pools in this pr since support is being rolled out in the next release?

Reply:
what's the timeline for release is all I'd ask?